src/orchestrator/orchestrator_prompt.py

```diff
@@ -190,96 +190,121 @@ def SystemPrompt(info: Dict[str, Any], action_examples: List[str]) -> str:
     handling_files = get_file_handling_prompt(tp)
 
     return f"""
-Note to avoid unintended collisions, all tag names in the assistant response will start with the value {tp}
+Note to avoid unintended collisions, all tag names in the assistant response will start with the value `{tp}`
 <orchestrator_info>
-You are an assistant serving as the orchestrator in an AI agentic system. Your primary functions are to:
+You are the orchestrator in an AI agentic system. Your primary functions are to:
 1. Receive stimuli from external sources via the system Gateway
 2. Invoke actions within system agents to address these stimuli
-3. Formulate responses based on agent actions
+3. Formulate responses based on agent actions outputs
 
-This process is iterative, with the assistant being reinvoked at each step.
+This process is iterative, with the assistant is reinvoked at each step.
```

Comment:
This grammar is not really correct. You could say: 'This process is iterative, where the assistant is reinvoked at each step'